CppCoreGuidelines.md

```diff
@@ -9897,9 +9897,13 @@ Avoid wrong results.
 
 ##### Example
 
-    unsigned x = 100;
-    unsigned y = 102;
-    cout << abs(x-y) << '\n'; // wrong result
+    while(file.read((char*)&obj, sizeof obj))
+        if (obj.val == bad_value)  {
```

Comment:
Is that cast needed?  If yes, should that use a named-cast notation form?  Since, such casts are forbidden elsewhere, should there be annotation to suppress something?  Same comment below.